modules/appeals_api/config/locales/en.yml

```diff
@@ -7,6 +7,9 @@ en:
         nod_not_found: "NoticeOfDisagreement with uuid %{id} not found."
         not_homeless_address_missing: "If not homeless, address must be provided: '/data/attributes/veteran/address'"
       evidence_submission:
+        expired_submission_window: "Evidence must be submitted within 90 days of Notice of Disagreement submission."
         max_dimensions_error: "File exceeds the maximum page dimensions."
         max_file_size_error: "File is too large."
         pdf_read_error: "File must be in PDF format."
+        veteran_id_mismatch: "Veteran ID does not match the Notice Of Disagreement Veteran ID."
+        unsupported_review_option: "The board review option selected on the corresponding notice of disagreement must be 'evidence_submission."
```

Comment:
Elsewhere you've capitalized Notice of Disagreement. Let's keep it consistent here, too.

Also looks like you're missing an end quote around `evidence_submission`.